src/net/dmytro/homework4/Cat.java

```diff
@@ -0,0 +1,32 @@
+package net.dmytro.homework4;
+
+public class Cat extends Animal {
+private String catName;
+private static int count = 0;
+
+    public Cat(String catName) {
+        this.catName = catName;
+    }
+
+    @Override
+    void ran(int maxRan) {
+        if (maxRan >= 0 ) {
```

Comment:
here is incorrect logic, if you try to ask your cat to run 40 000 km, it will die ) 

so maxRun is should be value cat can run as maximum distance, and you compare value from parameter (rename it as well) with max run. 

same with dogs